.snippets/code/get-started/start-building/batch/batch-jsonl-deepseek-v3-0324.py

```diff
@@ -0,0 +1,62 @@
+# Batch completions with the DeepSeek-V3-0324 model on Kluster.
+import os
+import json
+import kluster
+from typing import Dict, Any
+
+# 1. Initialize the Kluster SDK client
+#    Your API key can also be specified via KLUSTER_API_KEY environment variable
+client = kluster.Client(api_key="YOUR_API_KEY")
```

Comment:
This changed the way we used to do things. I like the old way better, we want to avoid people typing the API key, so please let's go back to the old API import mechanism